src/Model_Hydro/CPU_Hydro/CPU_dtSolver_HydroCFL.cpp

```diff
@@ -99,19 +108,32 @@ void CPU_dtSolver_HydroCFL  ( real g_dt_Array[], const real g_Flu_Array[][FLU_NI
          Bz2  = SQR( B[MAGZ] );
          B2   = Bx2 + By2 + Bz2;
          Emag = (real)0.5*B2;
-#        else
+#        elif ( !defined SRHD )
          Emag = NULL_REAL;
 #        endif
 
+
+#        ifdef SRHD
+         real Rho;
+         Hydro_Con2Pri( fluid, Pri,(real)NULL_REAL, NULL_BOOL, NULL_INT, NULL, NULL_BOOL,
+                        (real)NULL_REAL, NULL, NULL,
+                        EoS.GuessHTilde_FuncPtr, EoS.HTilde2Temp_FuncPtr,
+                        EoS.AuxArrayDevPtr_Flt, EoS.AuxArrayDevPtr_Int, EoS.Table, NULL, &LorentzFactor );
```

Comment:
- The input parameters are inconsistent with `Hydro_Con2Pri()`.
- Change `FLU_NIN_T` from `NCOMP_FLUID` to `NCOMP_TOTAL` in `Macro.h` for SRHD if it's really necessary to call `Hydro_Con2Pri()` here.

Reply:
Hi @hyschive, thanks--I realize that this has fallen behind `master`, unfortunately I have been tied up with a telescope proposal. Hoping to get back to this within a few weeks.

Reply:
@jzuhone Thanks! I just accidentally found this bug. Feel free to let me know if there is anything I can help.

Reply:
@hyschive does https://github.com/gamer-project/gamer/pull/185/commits/7ef64604e13f614cc26dffe6c2a7a7f4bb9a1385 address it?

Reply:
@jzuhone I just realized that I was checking a different version of the SRHD code. So some of my comments are actually incorrect. Sorry for the false alarm!

I'll add comments on those commits directly.